inst/include/Rcpp/String.h

```diff
@@ -395,9 +395,12 @@ namespace Rcpp {
             enc = encoding;
 
             if (valid) {
-                data = Rcpp_ReplaceObject(data, Rf_mkCharCE(Rf_translateCharUTF8(data), encoding));
+                const void* vmax = vmaxget();
```

Comment:
I noticed that translateCharUTF8() has, in the R sources:

https://github.com/wch/r-source/blob/61132afa07832f4e855da35773fa1341f1a8b481/src/main/sysutils.c#L992-L994

so it's recommended that we manage the stack after usage. Not sure if we should do this in this PR or just leave it out for now though.